### DISCUSSION TITLE
The documentation seems 50% complete

### DISCUSSION THREAD

**Discussion by nickjuntilla** [WalletConnect/walletconnect-monorepo#3419] [Developer Support]
I have a wallet and I would like to be able to use wallet connect protocol to allow DAPPs to connect to it. I've read through all the documentation for version 1 and 2 and it seems to mainly focus on connecting DAPPS, but now how to hook up a wallet served from a server. Am I missing something?

**Comment by xzilja** [comment]
There is new and extensive sample for v2 wallet available here
https://github.com/WalletConnect/web-examples/tree/main/wallets/react-wallet-v2

We are also focusing on documentation improvements (still early)
https://github.com/WalletConnect/walletconnect-docs/pull/160

I'd love to address your issue there, so to clarify by `served from a server` you mean wallet that is SSRd? Best thing to do in this case is to initiate and manage wallet connect on a client side as you would otherwise need to propagate all messaging through your server to the client (websockets) and it can get very complex

**Comment by egueiros** [comment]
Same situation seems to exist for Rect Native mobile wallets. No clear documentation, dependency lists or how to troubleshoot issues. But based on the comment above, it seems it's a work in progress.

**Comment by finessevanes** [comment]
@egueiros are these docs helpful? https://docs.walletconnect.com/2.0/javascript/guides/react-native

**Comment by nickjuntilla** [comment]
@0xAsimetriq Wallet connect is supposed to be a standard that other companies can use right? I'm not talking about as a company that connects to the wallet. I'm talking about being a company that provides wallet services to users and wants to be able to allow those users to use our private keys that we generate for them on other dapps. Does that make sense? We are a private key storage company that holds the private keys and data for our users. We want our users to be able to access DAPPs with the data we have stored for them. Is this the documentation that you are providing. It is unclear to me.

**Comment by nickjuntilla** [comment]
@0xAsimetriq I'm seeing this documentation and it looks like it responds to your requests to make sessions, but there is nothing about signing transaction or what the response object looks like. Is this the correct documentation? If a request comes in, what port and path does it respond to? Is there any documentation about what format the request comes in as and what shape the response should take? 

https://docs.walletconnect.com/2.0/javascript/web3wallet/wallet-usage ?

**Comment by glitch-txs** [comment]
Hi, we'll release a new docs version, please if you still have any questions please let me know
